xml/smt_client.xml

```diff
@@ -592,12 +592,31 @@
   <title>Registering SLE12 Clients against &smt; Test Environment</title>
 
   <para>
-   To configure a client to register against the test environment instead of
-   the production environment, modify <filename>/etc/SUSEConnect</filename> on
-   the client machine by setting:
+   To register a client in the testing environment, follow these steps:
   </para>
 
+  <procedure>
+   <step>
+    <para>
+     Delete a registration of the client host. Find more information in
+     <xref linkend="smt-registration-delete"/>
```

Comment:
This deletes the client on the server, but doesn't change the client itself I think. I would recommend to run `SUSEConnect --de-register` on the client.

Reply:
thanks, updated